### DISCUSSION TITLE
`react-native.config.js` is not processed, what step am I missing?

### DISCUSSION THREAD

**Discussion by r0ssIV** [invertase/react-native-firebase#6841] [Q&A]
Hi, not sure if that should go to issues or its just some little thing.

So I've installed `yarn add @react-native-firebase/app` and ran `pod install` and ran build several times, but I see that script phase `'[RNFB] Core Configuration'`, was not added to my target.

Looks like `node_modules/@react-native-firebase/app/react-native.config.js` is not processed at all.
As a result of it (or maybe not) am not getting any `firebase.json` config values.

Totally stuck on that part.

**Comment by mking-keeper** [comment]
Hi @rivaros , I ran into the same issue. It turns out this was happening because I have an abstract target. [This fix](https://github.com/invertase/react-native-firebase/issues/2800) worked for me
